ci/pipelines/deploy-to-test.yml

```diff
@@ -1103,6 +1143,7 @@ jobs:
     plan:
       - get: toolbox-ecr-registry-test
         trigger: true
+        passed: [build-and-push-toolbox-to-test-ecr]
```

Comment:
Ah - I guess this had no e2e tests so there was nothing to 'pass' before?

Reply:
Yeah